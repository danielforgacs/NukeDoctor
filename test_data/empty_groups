PreGroup {
    name KEEP
}
Group {
 name KEEP
}
 InsideGroup {
  name DROP_THIS
 }
end_group
PostGroup {
    name KEEP
}
