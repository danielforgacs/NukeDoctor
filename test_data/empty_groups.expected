PreGroup {
    name KEEP
}
Group {
 name KEEP
}
end_group
PostGroup {
    name KEEP
}
